master/internal/api_user.go

```diff
@@ -26,7 +26,7 @@ const determinedName = "determined"
 
 var (
 	errUserNotFound = status.Error(codes.NotFound, "user not found")
-	latinText       = regexp.MustCompile("[^\\p{Latin}\\p{N}\\s]")
+	latinText       = regexp.MustCompile("[^[:graph:]\\s]")
```

Comment:
I'm a bit conflicted about allowing any visible Unicode (particularly confusable letters) but I don't want this hindering the development. Makes sense to allow hyphen/dash characters.

Reply:
`:graph:` should only be ascii, no unicode: https://github.com/google/re2/wiki/Syntax#ascii